mitodl/mitxonline
# 4244: no error message is shown when education level is not specified in additional details modal

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/4244
https://github.com/mitodl/hq/issues/4272

### Description (What does it do?)
Documentation: https://docs.google.com/document/d/1W7jYty8HjkaYxSFXoUMF9ZYTOp8eJ4ssmINfgLhQ61Q/edit#heading=h.ie7st6eu9m5w



### How can this be tested?
Go through each form in the application and verify the following:
- Accessibility with a screen reader.
- https://github.com/mitodl/hq/issues/4244 is resolved.
- Requirements outlined in documentation are defined clearly and addressed: https://docs.google.com/document/d/1W7jYty8HjkaYxSFXoUMF9ZYTOp8eJ4ssmINfgLhQ61Q/edit#heading=h.ppw15bxqq3n7
